pyproject.toml

```diff
@@ -309,6 +309,7 @@ unfixable = [
 "setup.py" = ["INP001"]  # Part of configuration, not a package.
 "__init__.py" = ["E402", "F401", "F403"]
 "astropy/coordinates/angle_formats.py" = ["D415"]  # The docstrings are regexes.
+"astropy/coordinates/tests/test_geodetic_representations.py" = ["F401"]  # Unused import needed.
```

Comment:
Good that you found that! Though I think in this case a `# noqa: F401` at the import line is perhaps clearer... These lines really are more for cases were the code would otherwise just be littered with `noqa`.